package.json

```diff
@@ -2,6 +2,8 @@
   "name": "cdt-amalgamator",
   "description": "Amalgamator Debug Adapter from Eclipse CDT",
   "version": "0.0.3",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
Should this file be created somewhere? How is it made? If I do `yarn build` it doesn't end up in the `dist` directory.

Reply:
Hi @jonahgraham -san,
I think this file is needed to be able to import Amalgator into other repositories like renesas-gdb-vscode.

Reply:
Yes, but when I do a `yarn build` it isn't created. Is there something you need to do to create the file? Is the file in your `dist` dir? Do you need to enable `declaration` in tsconfig?